arcgis-ios-sdk-samples/Content Display Logic/Views/AboutView.swift

```diff
@@ -0,0 +1,145 @@
+// Copyright 2021 Esri.
```

Comment:
Shouldn't the copyright header be 2022 as this is going to be merged now?

Reply:
🎉